blog/blog.json

```diff
@@ -1,3 +1,4 @@
 {
-    "layout": "blog-layout.njk"
-}
\ No newline at end of file
+    "tags": ["posts"],
```

Comment:
Is this working properly? If so, we have remove the `posts` tag from the individual posts as it's no longer necessary (again, not necessary for this PR - but a nice way to clean things up)

Reply:
Removed them from individual posts, it required switching on [deep data merge](https://www.11ty.dev/docs/data-deep-merge/) but I think that makes sense to do.